examples/two-column-layout/datatsets/products_query_suggestions_settings.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Since these datasets are more doc-related, I think it could be a good idea indeed but I don't see any JSON assets in the doc repo, is that something we can do @sarahdayan?

Reply:
We don't really use the docs site as storage, any JSON dataset we link there is stored in https://github.com/algolia/datasets.

I think we should use this repository, even if this is tied to a given demo. We can create a sub-directory for it.

Reply:
I don't think people who want to use our demos clone the full repo. They either follow a tutorial on the docs and copy/paste bits of code, or they use CodeSandbox.

I understand the rationale of putting everything together, but I'm not a fan of growing the overall size of the repo for example's sample data. That's why we made [algolia/datasets](https://github.com/algolia/datasets) too—this repo isn't meant to be cloned, you can directly copy/paste files from there.

In the prospect of keeping these datasets here, if the goal is to show some sample data to help users build their own, we don't need such big files. Only one or two records per file should suffice. wdyt?

Reply:
I think we're already showing the record structure in the guide itself, if so, we could then follow your guidance and put the larger dataset into the dedicated repo - making it explicit in the project repo where to find those.